handler/api/api.go

```diff
@@ -285,6 +294,16 @@ func (s Server) Handler() http.Handler {
 					acl.CheckAdminAccess(),
 				).Delete("/{member}", collabs.HandleDelete(s.Users, s.Repos, s.Perms))
 			})
+
+			r.Route("/cards", func(r chi.Router) {
+				r.Get("/{build}", card.HandleFindAll(s.Builds, s.Card, s.Repos))
+				r.Get("/{build}/{stage}/{step}", card.HandleFind(s.Builds, s.Card, s.Stages, s.Steps, s.Repos))
+				r.Get("/{build}/{stage}/{step}/json", card.HandleFindData(s.Builds, s.Card, s.Stages, s.Steps, s.Repos))
+				r.With(
+					acl.AuthorizeAdmin,
```

Comment:
AuthorizeAdmin should be removed since it only allows a Drone system admin to remove a card.  The CheckAdminAccess will allow anyone with admin access to the repository to delete the card, which should be sufficient.

Reply:
removed